VimeoNetworking/Sources/VimeoClient.swift

```diff
@@ -469,10 +467,13 @@ extension VimeoClient
     ///
     /// - Parameters:
     ///   - appConfiguration: An AppConfiguration instance
-    public static func configureSharedClient(withAppConfiguration appConfiguration: AppConfiguration)
+    ///   - configureSessionManagerBlock: a block to configure the session manager
+    public static func configureSharedClient(withAppConfiguration appConfiguration: AppConfiguration, configureSessionManagerBlock: ConfigureSessionManagerBlock?)
     {
         self._sharedClient.configuration = appConfiguration
-        self._sharedClient.sessionManager = VimeoSessionManager.defaultSessionManager(appConfiguration: appConfiguration)
+        
+        let defaultSessionManager = VimeoSessionManager.defaultSessionManager(appConfiguration: appConfiguration, configureSessionManagerBlock: configureSessionManagerBlock)
+        self._sharedClient.sessionManager = defaultSessionManager
```

Comment:
It looks like we're already done with the previous session manager. Do we want to invalidate it here, before using a new session manager?